### DISCUSSION TITLE
Product Management

### DISCUSSION THREAD

**Discussion by ayushozha** [dailydotdev/daily#472] [Feature Requests]
Can you also include tags for Product Management? Here I believe it would be better to sort the articles that introduce products. Like the current Product Hunt products could be listed here. Also, a sub-category that enlists all the jobs for Product Management, especially remote opportunities.

**Comment by idoshamun** [comment]
Hi @Ayush10,

Our focus at the moment is software engineering with that said please feel free to submit new sources that you think might fit our community. Here's a guide on how to submit new sources, make sure to read our guidelines:
https://docs.daily.dev/docs/for-content-creators/suggest-new-source#how-to-submit-a-new-source-request
